pygmt/tests/test_clib_put_strings.py

```diff
@@ -0,0 +1,67 @@
+"""
+Test the functions that put string data into GMT.
+"""
+import numpy as np
+import numpy.testing as npt
+import pytest
+from packaging.version import Version
+
+from .. import clib
+from ..exceptions import GMTCLibError
+from ..helpers import GMTTempFile
+
+with clib.Session() as _lib:
+    gmt_version = Version(_lib.info["version"])
+
+
+@pytest.mark.xfail(
+    condition=gmt_version < Version("6.1.1"),
+    reason="GMT_Put_Strings only works for GMT 6.1.1 and above",
+)
+def test_put_strings():
+    "Check that assigning a numpy array of dtype str to a dataset works"
```

Comment:
Temporarily using xfail so all the tests will be green. If it looks alright, we can merge this in first and remove these `xfail`s after bumping the minimum required GMT version to 6.1.1. Can also do it when we refactor `text` to use `put_strings` :smile:

Reply:
Xfails have been removed in #577 at c0bb8be065870879128f3f2804c161746e0dc9f7.